src/modules/fancyzones/editor/FancyZonesEditor/GridZone.xaml.cs

```diff
@@ -101,7 +101,16 @@ private bool IsVerticalSplit
 
         private int SplitterThickness
         {
-            get { return Math.Max(((App)Application.Current).ZoneSettings.Spacing, 5); }
+            get
+            {
+                Settings settings = ((App)Application.Current).ZoneSettings;
+                if (!settings.ShowSpacing)
+                {
+                    return 1;
+                }
+
+                return Math.Max(settings.Spacing, 5);
```

Comment:
Since we are using the 1px thickness for 0 spacing, let's also update the "5" minimum to "1".

Reply:
Sure